lib/entities/request/index.ts

```diff
@@ -59,6 +60,7 @@ export interface QuoteRequest {
   routingType: RoutingType;
   info: QuoteRequestInfo;
   config: RoutingConfig;
+  headers: APIGatewayProxyEventHeaders;
```

Comment:
It's probably better if we make `headers` an optional field, so that not every concrete request has to inherit it. Only classic needs the request source for now.

Reply:
I had it that way initially but then I thought that it might get confusing if we ever need to include in the other request types, so I wanted to keep this as easier to understand as possible, so right now all the existing request will ahve the headers available to them but they are not using them

Reply:
Also I think it's better if we don't mix our request field with the AWS native headers type.

Reply:
gotcha, the underlying type is:
```
export interface APIGatewayProxyEventHeaders {
    [name: string]: string | undefined;
}
```

I could just copy and paste it, but I assumed that passing the headers using the same type was okay